PROJETOS/20251105-github-actions-workflow-modifier/convert_to_choice.py

```
@@ -0,0 +1,88 @@
+    parser.add_argument('workflow', help='Arquivo YAML do workflow')
+    parser.add_argument('-o', '--output', help='Arquivo de saída (padrão: inplace)')
+    parser.add_argument('-v', '--verbose', action='store_true')
+
+    args = parser.parse_args()
+
+    yaml = YAML()
+    yaml.preserve_quotes = True
+    yaml.width = 4096
+
+    print(f"📖 Lendo: {args.workflow}")
+    with open(args.workflow, 'r') as f:
```

Ruff (0.14.3)

Unnecessary mode argument

Remove mode argument

(UP015)